client/extensions/woocommerce/rest-api-client/registry/index.js

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ApiClient from '../api-client';
+
+const debug = debugFactory( 'rest-api-client:registry' );
+const _apiTypes = {};
+const _apiClients = new Map();
+
+// TODO: Unit test this.
+export function registerApiType( name, apiType, apiTypes = _apiTypes ) {
```

Comment:
What exactly is apiType? It sounds like it should be a string name, but I see above it's the `createWcApi` object. There's a lot of `apiType*` in this file, and maybe these could have better names - some documentation might help with figuring out clearer names.

Reply:
Yeah, there's ambiguity here. Maybe the string should be `apiName`?

Reply:
`name` is fine, but if `apiType` is actually a collection of "stuff", it's not a type itself - the type is "woocommerce" (in my reading). This argument could be … a `client`, or `handlers` … I don't think you need to prefix everything with API, either -- that's obvious from the file itself, and function names (which since they're used externally, can keep the `Api`).

Reply:
So, the `apiType` object would represent everything one needs to use the WooCommerce API, but it wouldn't be keyed to a specific site, so what would you recommend naming that?